anvil/observables.py

```diff
@@ -111,7 +119,7 @@ def magnetic_susceptibility(magnetization, abs_magnetization_squared):
 
 
 def magnetization_series(configs):
-    return configs.sum(axis=1)
+    return configs.sum(axis=1).numpy()
```

Comment:
would it make more sense to just cast the configs numpy earlier instead of multiple times on the observables?

Reply:
in other words, do we ever need `configs` to be a tensor?

Reply:
I'd rather just leave this niggle until we're messing around inside `sample.py` anyway, trying to generate configurations from each layer, if that's alright.